src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -71,10 +74,9 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(
     const quotes = useMemo(
       () =>
         sortedQuotes.map((quoteData, i) => {
-          const { index } = quoteData
+          const { quote } = quoteData
 
-          // TODO(woodenfurniture): use quote ID when we want to support multiple quotes per swapper
-          const isActive = activeQuoteIndex === index
+          const isActive = activeQuoteIndex !== undefined && activeQuoteIndex === quote?.id
```

Comment:
Do we want to change the `activeQuoteIndex` to match the new `ID` vernacular as well?

Reply:
20321f868e3dae2476911776307a5ceb6b8859d0